packages/cli/tests/images/build.js

```diff
@@ -55,10 +55,12 @@ exports.prerender.heads.home = `
 	<link rel="preload" href="\\/route-home\\.chunk\\.\\w{5}\\.js" as="script">
 	<link rel="preload" href="\\/route-home\\.chunk\\.\\w{5}\\.css" as="style">
 	<link rel="shortcut icon" href="\\/favicon\\.ico">
-	<link href=\\"/bundle.\\w{5}.css\\" rel=\\"preload\\" as=\\"style\\" onload=\\"this.rel='stylesheet'\\"><noscript>
-		<link rel=\\"stylesheet\\" href="\\/bundle.\\w{5}.css"><\\/noscript>
-	<style>html{padding:0;}<\\/style>
-<\\/head>
+	<style>html{padding:0}</style>
```

Comment:
whoops! I have fixed the same test images in #695 😅

Reply:
Oof